src/types/Error.ts

```diff
@@ -0,0 +1,8 @@
+export enum Error {
```

Comment:
Consider renaming the enum 'Error' to 'ErrorType' to avoid confusion with the built-in 'Error' object in JavaScript.